fairmq/nanomsg/FairMQTransportFactoryNN.cxx

```diff
@@ -63,7 +63,8 @@ FairMQPollerPtr FairMQTransportFactoryNN::CreatePoller(const FairMQSocket& cmdSo
 
 void FairMQTransportFactoryNN::Shutdown()
 {
-    nn_term();
+    // nn_term();
+    // see https://www.freelists.org/post/nanomsg/Getting-rid-of-nn-init-and-nn-term,8
 }
```

Comment:
Yeah, I saw it, also didn't look deeper how they use the flag. But anyway, the fact that it is documented as optional for something that we solve differently makes it fine to remove. None of our sockets hang if device wants to exit. This call is just a secondary safety. And the fact that they have it global for all sockets will give us trouble if we need more than one context.